#,Node,Feature,TestCase,Result,ResultType,ExecutedOn,Description,Function,Module,Devices,KnownIssue
1,gw2,Regression,ft_add_unknownvlan_interface,Pass,Mapped,2023-12-31 03:11:33,Test case passed,test_ft_add_unknownvlan_interface,switching/test_vlan.py,"V1, V2",
2,gw2,Regression,ft_vlan_delete_with_member,Pass,Mapped,2023-12-31 03:11:46,Test case passed,test_ft_vlan_delete_with_member,switching/test_vlan.py,"V1, V2",
3,gw2,Regression,ft_vlan_trunk_tagged,Pass,Mapped,2023-12-31 03:13:11,Test case passed,test_ft_vlan_trunk_tagged,switching/test_vlan.py,"V1, V2",
4,gw2,Regression,ft_show_vlan_brief,Pass,Mapped,2023-12-31 03:13:11,Test case passed,test_ft_vlan_trunk_tagged,switching/test_vlan.py,"V1, V2",
5,gw3,Regression,FtOpSoSwLagFn042,Fail,Mapped,2023-12-31 03:13:31,"Portchannel PortChannel7 on V4 with member ['Ethernet48', 'Ethernet52'] verification failed",test_ft_portchannel_with_vlan_variations,switching/test_portchannel.py,"V3, V4",
6,gw3,Regression,FtOpSoSwLagFn041,Fail,Mapped,2023-12-31 03:13:31,"Portchannel PortChannel7 on V4 with member ['Ethernet48', 'Ethernet52'] verification failed",test_ft_portchannel_with_vlan_variations,switching/test_portchannel.py,"V3, V4",
7,gw3,PortChannel Config Enhancements,FtOpSoSwPominFn007,ConfigFail,Mapped,2023-12-31 03:14:09,pre-test configuration failed Invalid error code PortChannel Verification Failed : unknown message identifier 'PortChannel Verification Failed' @5430,test_ft_verify_min_links_functionality_by_flapping_all_member_ports_007,switching/test_portchannel.py,"V3, V4",
8,gw2,warmboot,ft_reboot_fdb_fast_reboot,ScriptError,Mapped,2023-12-31 03:18:23,Exception TypeError 'NoneType' object is not subscriptable in file /apis/system/basic.py at line 1389,test_ft_vlan_save_config_warm_and_fast_reboot,switching/test_vlan.py,"V1, V2",
9,gw2,warmboot,ft_max_vlan_save_reload,ScriptError,Mapped,2023-12-31 03:18:23,Exception TypeError 'NoneType' object is not subscriptable in file /apis/system/basic.py at line 1389,test_ft_vlan_save_config_warm_and_fast_reboot,switching/test_vlan.py,"V1, V2",
10,gw2,warmboot,FtOpSoSysFRFn005,ScriptError,Mapped,2023-12-31 03:18:23,Exception TypeError 'NoneType' object is not subscriptable in file /apis/system/basic.py at line 1389,test_ft_vlan_save_config_warm_and_fast_reboot,switching/test_vlan.py,"V1, V2",
11,gw3,Regression,ft_lldp_LocManAddrLen,Fail,Mapped,2023-12-31 03:18:34,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrLen,system/test_lldp.py,"V3, V4",
12,gw3,Regression,ft_lldp_LocManAddrlfld,Fail,Mapped,2023-12-31 03:18:34,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrlfld,system/test_lldp.py,"V3, V4",
13,gw3,Regression,ft_lldp_LocManAddrEntry,Fail,Mapped,2023-12-31 03:18:35,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrEntry,system/test_lldp.py,"V3, V4",
14,gw3,Regression,ft_lldp_ConfigManAddrEntry,Fail,Mapped,2023-12-31 03:18:35,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_ConfigManAddrEntry,system/test_lldp.py,"V3, V4",
15,gw3,Regression,ft_lldp_lldplocportid,Fail,Mapped,2023-12-31 03:18:35,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocportid,system/test_lldp.py,"V3, V4",
16,gw3,Regression,ft_lldp_lldplocsysname,Fail,Mapped,2023-12-31 03:18:36,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocsysname,system/test_lldp.py,"V3, V4",
17,gw3,Regression,ft_lldp_lldplocsysdesc,Fail,Mapped,2023-12-31 03:18:36,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocsysdesc,system/test_lldp.py,"V3, V4",
18,gw3,SNMP Agent Optimization,ft_snmp_lldp_prem_manaddr_table,Fail,Mapped,2023-12-31 03:18:36,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_rem_man_addr_table,system/test_lldp.py,"V3, V4",
19,gw3,Regression,FtOpSoSwlldpFn011,Fail,Mapped,2023-12-31 03:18:37,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
20,gw3,Regression,FtOpSoSwlldpFn010,Fail,Mapped,2023-12-31 03:18:37,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
21,gw3,Regression,FtOpSoSwlldpFn009,Fail,Mapped,2023-12-31 03:18:37,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
22,gw3,Regression,FtOpSoSwlldpFn008,Fail,Mapped,2023-12-31 03:18:37,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
23,gw3,Regression,FtOpSoSwlldpFn004,Fail,Mapped,2023-12-31 03:18:37,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
24,gw3,Regression,FtOpSoSwlldpFn013,Fail,Mapped,2023-12-31 03:18:37,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_docker_restart,system/test_lldp.py,"V3, V4",
25,gw3,Regression,lldp_verify_counters,Fail,Mapped,2023-12-31 03:18:37,module configuration failed. LLDP neighbors information is not found after polling @5445,test_lldp_verify_counters,system/test_lldp.py,"V3, V4",
26,gw3,Regression,ft_lldp_with_remote_server,Fail,Mapped,2023-12-31 03:18:38,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_with_remote_server,system/test_lldp.py,"V3, V4",
27,gw3,Regression,FtOpSoSyInIpFn002,Fail,Mapped,2023-12-31 03:23:13,interface is down on DUT ['Ethernet48'],test_ft_port_fn_verify_shut_noshut,system/test_interface.py,"V3, V4",
28,gw2,Regression,ft_ssh_service_disable,Pass,Mapped,2023-12-31 03:24:19,Test case passed,test_ft_ssh_service_disable,system/test_ssh.py,"V1, V2",
29,gw7,Instrumentation,ft_sf_wm_telemetry_interval,Pass,Mapped,2023-12-31 03:30:54,verification of configured telemetry interval and reset to default interval is successful,test_ft_watermark_telemetry_interval,system/test_snapshot.py,V4,
30,gw7,Instrumentation Enhancements,ft_sf_snapshot_tam_Error_Logs,Fail,SubTest,2023-12-31 03:30:54,Error logs are getting generated on the device,test_ft_watermark_telemetry_interval,system/test_snapshot.py,V4,
31,gw2,Regression,ft_ssh_add_user_verify,Fail,Mapped,2023-12-31 03:33:24,Test case Failed.,test_ft_ssh_add_user_verify,system/test_ssh.py,"V1, V2",
32,gw2,Control Plane ACL,test_ft_controlplane_acl_service_sshv4,Pass,SubTest,2023-12-31 03:33:24,SSH failed,test_ft_ssh_add_user_verify,system/test_ssh.py,"V1, V2",
33,gw2,Control Plane ACL,test_ft_controlplane_acl_service_sshv6,Pass,SubTest,2023-12-31 03:33:24,SSH failed,test_ft_ssh_add_user_verify,system/test_ssh.py,"V1, V2",
34,gw2,Control Plane ACL,test_ft_controlplane_acl_service_snmp,Fail,SubTest,2023-12-31 03:33:24,SNMP output is not found after walk of with control plane ACL service SNMP after reboot OID,test_ft_ssh_add_user_verify,system/test_ssh.py,"V1, V2",
35,gw2,Regression,CETA_SONIC_67575,Pass,Mapped,2023-12-31 03:33:46,Test case passed,test_ft_verify_logs_at_the_time_of_ssh_login_success,system/test_ssh.py,"V1, V2",
36,gw7,Regression,ft_tacacsauth_enable_pamfile_created,Fail,Mapped,2023-12-31 03:34:29,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V4,
37,gw7,Regression,ft_tacacs_ssh_login,Fail,Mapped,2023-12-31 03:34:29,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V4,
38,gw7,Regression,ft_tacacs_single_server_for_auth,Fail,Mapped,2023-12-31 03:34:29,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V4,
39,gw7,Regression,ft_tacacs_default_login,Fail,Mapped,2023-12-31 03:34:29,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V4,
40,gw7,Regression,executing_server_key_in_appropriate_mode,Fail,Mapped,2023-12-31 03:34:29,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V4,
41,gw7,Regression,auth_shared_keys ,Fail,Mapped,2023-12-31 03:34:29,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V4,
42,gw5,Regression,ft_ntp_message_log_display_with_correct_time,Pass,Mapped,2023-12-31 03:35:42,Test case passed,test_ft_ntp_disable_enable_with_message_log,system/test_ntp.py,V3,
43,gw5,Regression,ft_ntp_disable_ntp_enable_ntp,Pass,Mapped,2023-12-31 03:35:42,Test case passed,test_ft_ntp_disable_enable_with_message_log,system/test_ntp.py,V3,
44,gw5,Regression,ft_ntp_existsconfig,Pass,Mapped,2023-12-31 03:35:52,Test case passed,test_ntp_exists_config,system/test_ntp.py,V3,
45,gw7,Regression,ft_snmp_sysName,Fail,Mapped,2023-12-31 03:39:25,sysname output is not in sync between system and snmp,test_ft_snmp_sysName,system/test_snmp.py,V4,
46,gw7,Regression,ft_snmp_test_syUpTime,Pass,Mapped,2023-12-31 03:39:31,Test case passed,test_ft_snmp_test_syUpTime,system/test_snmp.py,V4,
47,gw7,Regression,ft_snmp_sysDescr,Pass,Mapped,2023-12-31 03:39:38,Test case passed,test_ft_snmp_sysDescr,system/test_snmp.py,V4,
48,gw7,Regression,FtOpSoSySnFn008,Pass,Mapped,2023-12-31 03:39:49,Test case passed,test_ft_snmp_mib_2,system/test_snmp.py,V4,
49,gw7,Regression,FtOpSoSySnFn009,Pass,Mapped,2023-12-31 03:39:53,Test case passed,test_ft_snmp_if_mib_all,system/test_snmp.py,V4,
50,gw7,Regression,FtOpSoSySnFn010,Pass,Mapped,2023-12-31 03:39:58,Test case passed,test_ft_snmp_entity_mib_all,system/test_snmp.py,V4,
51,gw7,Regression,FtOpSoSySnFn013,Pass,Mapped,2023-12-31 03:40:03,Test case passed,test_ft_snmp_dot1q_dot1db_mib,system/test_snmp.py,V4,
52,gw7,Regression,FtOpSoSySnFn012,Pass,Mapped,2023-12-31 03:40:03,Test case passed,test_ft_snmp_dot1q_dot1db_mib,system/test_snmp.py,V4,
53,gw7,Regression,FtOpSoSySnFn015,Pass,Mapped,2023-12-31 03:40:19,Test case passed,test_ft_snmp_root_node_walk,system/test_snmp.py,V4,
54,gw7,Regression,FtOpSoSySnFn016,Pass,Mapped,2023-12-31 03:40:30,Test case passed,test_ft_snmp_ipAddressRowStatus_ipv6,system/test_snmp.py,V4,
55,gw7,Regression,FtOpSoSySnFn017,Pass,Mapped,2023-12-31 03:40:34,Test case passed,test_ft_snmp_ipAddressStorageType_ipv6,system/test_snmp.py,V4,
56,gw7,Regression,FtOpSoSySnFn020,Pass,Mapped,2023-12-31 03:40:39,Test case passed,test_ft_snmp_ipv6_If_Forward_default_HopLimit,system/test_snmp.py,V4,
57,gw7,Regression,FtOpSoSySnFn021,Pass,Mapped,2023-12-31 03:40:43,Test case passed,test_ft_snmp_ipv6scope_index_table,system/test_snmp.py,V4,
58,gw7,Regression,test_snmp_ip_forward_table,Pass,Mapped,2023-12-31 03:40:50,Test case passed,test_ft_snmp_ip_forward_table,system/test_snmp.py,V4,
59,gw7,SNMP Agent Optimization,ft_snmp_ifXTable,Pass,Mapped,2023-12-31 03:40:56,Test case passed,test_ft_snmp_ifx_table,system/test_snmp.py,V4,
60,gw7,SNMP Agent Optimization,ft_snmp_ip_System_Stats_Table,Pass,Mapped,2023-12-31 03:41:00,Test case passed,test_ft_snmp_ip_System_Stats_Table,system/test_snmp.py,V4,
61,gw7,SNMP Agent Optimization,ft_snmp_ip_IfStats_Table,Pass,Mapped,2023-12-31 03:41:05,Test case passed,test_ft_snmp_ip_IfStats_Table,system/test_snmp.py,V4,
62,gw7,SNMP Agent Optimization,ft_snmp_ip_Address_Table,Pass,Mapped,2023-12-31 03:41:10,Test case passed,test_ft_snmp_ip_Address_Table,system/test_snmp.py,V4,
63,gw7,SNMP Agent Optimization,ft_snmp_ip_NetToPhysical_Table,Pass,Mapped,2023-12-31 03:41:14,Test case passed,test_ft_snmp_ip_NetToPhysical_Table,system/test_snmp.py,V4,
64,gw7,SNMP Agent Optimization,ft_snmp_icmp_Msgs,Pass,Mapped,2023-12-31 03:41:18,Test case passed,test_ft_snmp_icmp_Msgs,system/test_snmp.py,V4,
65,gw7,SNMP Agent Optimization,ft_snmp_tcp_mib,Pass,Mapped,2023-12-31 03:41:40,Test case passed,test_ft_snmp_tcp_mib,system/test_snmp.py,V4,
66,gw7,SNMP Agent Optimization,ft_snmp_udp_mib,Pass,Mapped,2023-12-31 03:41:44,Test case passed,test_ft_snmp_udp_mib,system/test_snmp.py,V4,
67,gw7,SNMP Agent Optimization,ft_snmp_snmpv2_mib,Pass,Mapped,2023-12-31 03:41:49,Test case passed,test_ft_snmp_snmpv2_mib,system/test_snmp.py,V4,
68,gw7,SNMP Agent Optimization,ft_snmp_host_resource_mib,Pass,Mapped,2023-12-31 03:41:53,Test case passed,test_ft_snmp_host_resource_mib,system/test_snmp.py,V4,
69,gw7,SNMP Agent Optimization,ft_snmp_framework_mib,Pass,Mapped,2023-12-31 03:41:58,Test case passed,test_ft_snmp_framework_mib,system/test_snmp.py,V4,
70,gw7,SNMP Agent Optimization,ft_snmp_mpd_mib,Pass,Mapped,2023-12-31 03:42:02,Test case passed,test_ft_snmp_mpd_mib,system/test_snmp.py,V4,
71,gw7,SNMP Agent Optimization,ft_snmp_target_mib,Pass,Mapped,2023-12-31 03:42:07,Test case passed,test_ft_snmp_target_mib,system/test_snmp.py,V4,
72,gw7,SNMP Agent Optimization,ft_snmp_notification_mib,Pass,Mapped,2023-12-31 03:42:12,Test case passed,test_ft_snmp_notification_mib,system/test_snmp.py,V4,
73,gw7,SNMP Agent Optimization,ft_snmp_user_based_sm_mib,Pass,Mapped,2023-12-31 03:42:16,Test case passed,test_ft_snmp_user_based_sm_mib,system/test_snmp.py,V4,
74,gw7,SNMP Agent Optimization,ft_snmp_view_based_acm_mib,Pass,Mapped,2023-12-31 03:42:21,Test case passed,test_ft_snmp_view_based_acm_mib,system/test_snmp.py,V4,
75,gw7,SNMP Agent Optimization,ft_snmp_ent_physical_table,Pass,Mapped,2023-12-31 03:42:25,Test case passed,test_ft_snmp_ent_physical_table,system/test_snmp.py,V4,
76,gw7,SNMP Agent Optimization,ft_snmp_dot3_stats_table,Pass,Mapped,2023-12-31 03:42:30,Test case passed,test_ft_snmp_dot3_stats_table,system/test_snmp.py,V4,
77,gw7,SNMP Agent Optimization,ft_net_snmp_agent_mib,Pass,Mapped,2023-12-31 03:42:35,Test case passed,test_ft_net_snmp_agent_mib,system/test_snmp.py,V4,
78,gw7,SNMP Agent Optimization,ft_net_snmp_vacm_mib,Pass,Mapped,2023-12-31 03:42:39,Test case passed,test_ft_net_snmp_vacm_mib,system/test_snmp.py,V4,
79,gw7,SNMP Agent Optimization,ft_snmp_ucd_diskio_mib,Pass,Mapped,2023-12-31 03:42:44,Test case passed,test_ft_snmp_ucd_diskio_mib,system/test_snmp.py,V4,
80,gw7,SNMP Agent Optimization,ft_snmp_ucd_la_table,Pass,Mapped,2023-12-31 03:42:48,Test case passed,test_ft_snmp_ucd_la_table,system/test_snmp.py,V4,
81,gw7,SNMP Agent Optimization,ft_snmp_ucd_system_stats,Pass,Mapped,2023-12-31 03:42:53,Test case passed,test_ft_snmp_ucd_system_stats,system/test_snmp.py,V4,
82,gw7,SNMP MIB and Traps,SNMPQBr003,Fail,Mapped,2023-12-31 03:42:59,SNMP output is not found after walk of dot1qTpFdbPort OID,test_ft_snmp_dot1q_tp_fdb_port,system/test_snmp.py,V4,
83,gw7,SNMP MIB and Traps,SNMPQBr002,Pass,Mapped,2023-12-31 03:43:04,Test case passed,test_ft_snmp_dot1q_tp_fdb_address,system/test_snmp.py,V4,
84,gw7,In-Memory Debug Log,FtOpSoSysImLogFn012,EnvFail,Mapped,2023-12-31 03:43:22,Test case Failed Couldn't get the snmptrapd process details from snmp trap server. Please check the snmp trap server process in the server.,test_ft_snmp_coldstart_trap,system/test_snmp.py,V4,
85,gw7,SNMP MIB and Traps,SNMPTrap004,EnvFail,Mapped,2023-12-31 03:43:22,Test case Failed Couldn't get the snmptrapd process details from snmp trap server. Please check the snmp trap server process in the server.,test_ft_snmp_coldstart_trap,system/test_snmp.py,V4,
86,gw7,Regression,FtOpSoSySnFn023,Fail,Mapped,2023-12-31 03:43:38,SNMP service is not up,test_ft_snmp_docker_restart,system/test_snmp.py,V4,
